rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -207,6 +207,7 @@ def constant_value_to_py(type_, value):
 
 def escape_string(s):
     s = s.replace('\\', '\\\\')
+    s = s.replace('"', '\\"')
```

Comment:
This escape is not necessary when the template always quotes string with single quotes.